tools/wake/main.cpp

```diff
@@ -639,7 +652,7 @@ int main(int argc, char **argv) {
   }
 
   if (is_db_inspection) {
-    inspect_database(clo, db, wake_cwd);
```

Comment:
This argument was just dead code already?

Reply:
Yeah it was already dead code